java/src/test/java/org/rocksdb/HyperClockCacheTest.java

```diff
@@ -0,0 +1,25 @@
+package org.rocksdb;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class HyperClockCacheTest {
+  @Rule public TemporaryFolder dbFolder = new TemporaryFolder();
+
+  @Test
+  public void newHyperClockCache() throws RocksDBException {
+    RocksDB.loadLibrary();
+    try (Cache cache = new HyperClockCache(1024 * 1024, 0, 8, false)) {
+      BlockBasedTableConfig tableConfing = new BlockBasedTableConfig();
+      tableConfing.setBlockCache(cache);
+      try (Options options = new Options()) {
+        options.setTableFormatConfig(tableConfing);
+        options.setCreateIfMissing(true);
+        try (RocksDB db = RocksDB.open(options, dbFolder.getRoot().getAbsolutePath())) {
+          db.put("testKey".getBytes(), "testData".getBytes());

```

Comment:
Would it be a good idea to call the `cache.getUsage()` and `cache.getPinnedUsage()` methods of the `Cache` superclass, as the `LRUCacheTest` does ? Just to sanity check the calls and give some confidence that the cache object is valid

Reply:
Good point. It looks like it works. Do you think we can put it inside assert? 
```java
assertThat(cache.getUsage()).isNotZero();
assertThat(cache.getPinnedUsage()).isNotZero();
```

Reply:
Yes. See LRUCacheTest.java